sfge/src/main/java/com/salesforce/rules/UnusedMethodRule.java

```diff
@@ -251,28 +410,45 @@ private List<Violation> convertMethodsToViolations() {
                 .collect(Collectors.toList());
     }
 
-    private static final class InternalCallValidator
-            extends TypedVertexVisitor.DefaultNoOp<Boolean> {
-        private final MethodVertex invokedMethod;
+    /** Base class for call validators. */
+    private static class CallValidator extends TypedVertexVisitor.DefaultNoOp<Boolean> {
```

Comment:
Can each of these visitors be in their own public class? And all of unusedMethod-specific classes can be in their own package. As a person new to the code, viewing them as individual classes helps with readability. This can surely wait till the next iteration.